extensions/amp-iframe/1.0/component.js

```diff
@@ -26,23 +30,96 @@ export function Iframe({
   allowFullScreen,
   allowPaymentRequest,
   allowTransparency,
-  onLoadCallback = NOOP,
+  onLoad = NOOP,
   referrerPolicy,
+  requestResize,
   sandbox,
   src,
   srcdoc,
   ...rest
 }) {
+  const iframeRef = useRef();
+  const dataRef = useRef(null);
+  const isIntersectingRef = useRef(false);
```

Comment:
`attemptResize` can be called before the `IntersectionObserver` starts observing the component if an event is fired immediately, such as in the "Resizable iframe in viewport" Preact Storybook example, revealing that the `!isIntersectingRef.current` condition can be met unintentionally. Let's either:
- Default this to be `true` since a false positive behaves more safely (no CLS) than a false negative
- Default this to be `null` and update the `!isIntersectingRef.current` check to be `isIntersectingRef.current === false`

Reply:
Good point. Done (also added comment explaining this scenario).